docs/fsharp/what-is-fsharp.md

```diff
@@ -1,31 +1,29 @@
 ---
 title: What is F#
-description: Learn about what the F# programming language is and what F# programming is like. Learn about rich data types, functions, and how they fit together.
+description: Learn about what the F# programming language is and what F# programming is like. Learn about rich types, functions, and how they fit together.
 ms.date: 08/03/2018
 ---
 # What is F\#
 
-F# is a functional programming language that makes it easy to write correct and maintainable code.
+F# is an open-source, cross-platform, interoperable programming language for writing succinct, robust and performant code.
 
-F# programming primarily involves defining types and functions that are type-inferred and generalized automatically. This allows your focus to remain on the problem domain and manipulating its data, rather than the details of programming.
+F# programming is data-oriented and code involves transforming data with functions. An F# program is often
+a succinct definition of types that model a domain and functions that implement your chosen transformations.
+Your focus remains on your problem domain, rather than the details of programming.
 
 ```fsharp
 open System // Gets access to functionality in System namespace.
 
 // Defines a function that takes a name and produces a greeting.
-let getGreeting name = $"Hello, {name}! Isn't F# great?"
+let getGreeting name = $"Hello, {name}"
 
-[<EntryPoint>]
-let main args =
-    // Defines a list of names
-    let names = [ "Don"; "Julia"; "Xi" ]
+// Defines a list of names
+let names = [ "Peter"; "Julia"; "Xi" ]
```

Comment:
Glad you left in one of my little easter eggs still 😉

Reply:
ha yes